cmake/sca/eclair/sca.cmake

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2023, BUGSENG Srl
+
+find_program(ECLAIR_ENV eclair_env REQUIRED)
+message(STATUS "Found eclair_env: ${ECLAIR_ENV}")
+
+find_program(ECLAIR_REPORT eclair_report REQUIRED)
+message(STATUS "Found eclair_report: ${ECLAIR_REPORT}")
+
+# ECLAIR Settings
+set(ECLAIR_PROJECT_NAME "Zephyr-${BOARD}")
```

Comment:
with the new hw model in place, should this then be `Zephyr-${BOARD}${BOARD_QUALIFIERS}` ?

Non-blocking comment.

Reply:
Yes with the new model in place this will be switched to ${BOARD}${BOARD_QUALIFIERS}

Reply:
Time to switch this over now that HWMv2 is the norm